### DISCUSSION TITLE
Version that doesn't use active borders.

### DISCUSSION THREAD

**Discussion by Twystedgamer** [tryptech/obs-zoom-and-follow#12] [Ideas]
I'm a little new to the programming industry; I apologize if what I'm asking doesn't make sense. But I've been trying to tweak the code to allow the crop to directly show the cursor with no "following," just directly zooming into the mouse at all times. Maybe this'd be a little ambitious to use without any zooming but if it can run at the refresh rate it should be fine. Or a version that has smoothing but directly follows the mouse with no "active border." Any ideas on how to do that? It's hard reading code that's not my own so I haven't gained much ground for the last few hours.

Edit: After some time I figured it out and I think I found the cause of the jitter, the active zone edges overlap each other causing some nasty footage without a ton of smoothing. I fixed it on the modified version and it directly follows the mouse with no problem whatso-ever!

**Comment by Twystedgamer** [comment]
And possibly make this an addition to the main script by adding a checkbox of some sort to directly follow the mouse no matter what if it seems to be a good feature

**Comment by tryptech** [comment]
Glad to hear you got it working. I believe that setting the active border to 0.5 from the default of 0.15 should achieve what you're describing, but you got it working anyway.

**Comment by Twystedgamer** [comment]
I tried messing with those settings and the zoom would just keep spazing out, so I modified the script to just do what I wanted; The zoom follows the mouse directly with no active border

**Reply by tryptech** [reply]
Alright, thanks for letting me know. I'll probably set it as a toggle to ignore the border.
